io.itch.drwhut.TabletopClub.json

```diff
@@ -0,0 +1,57 @@
+{
+    "app-id": "io.itch.drwhut.TabletopClub",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "22.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "tabletop-club",
+    "finish-args": [
+        "--socket=x11",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--share=ipc",
+        "--device=dri",
+        "--socket=pulseaudio",
+        "--share=network",
+        "--filesystem=xdg-desktop/TabletopClub:ro",
+        "--filesystem=xdg-documents/TabletopClub:create",
+        "--filesystem=xdg-download/TabletopClub:ro"
```

Comment:
```suggestion
        "--filesystem=xdg-documents/TabletopClub:create",
```

Reply:
Fair enough, I shall implement this change.